.github/workflows/test-only.yml

```diff
@@ -108,8 +108,11 @@ jobs:
           # invert the check (it should be not 0/not running)
           docker ps
           docker logs sig-test
-          docker inspect --format '{{json .State.Running}}' sig-test
-          if [ $? -eq 0 ]
+          docker inspect --format '{{json .State.Running}}' sig-test | grep false
```

Comment:
May I ask you to run this code below?
```
docker inspect --format '{{json .State.Running}}' container_alive_name | grep false
echo $?
docker inspect --format '{{json .State.Running}}' container_dead_name | grep false
echo $?
```

Reply:
Also please check the latest test log too.
https://github.com/dgtlmoon/changedetection.io/actions/runs/6518429056/job/17703847214?pr=1850#step:14:101
https://github.com/dgtlmoon/changedetection.io/actions/runs/6518429056/job/17703847214?pr=1850#step:14:142

Reply:
```
dgtlmoon:~$ docker inspect --format '{{json .State.Running}}' xxxx  | grep false
dgtlmoon@:~$ echo $?
1
dgtlmoon@:~$ docker inspect --format '{{json .State.Running}}' xxxx  | grep false
Error: No such object: xxxx
dgtlmoon@:~$ echo $?
1
```

Reply:
this is not a good way to check i think, best to use whats in my PR

Reply:
https://unix.stackexchange.com/a/501129/546660
> echo $? will return the exit status of last command.

For your grep in first command, it shows 1. Because there is no 'false' in stdout.
For your grep in second command, it shows 1. Because there is no 'false' in stdout.

So in this case, my new test will return 1 and the test will fail as we expect.
because my new test uses `if [ $? -ne 0 ]`(https://stackoverflow.com/a/17689530/20307768) instead of `if [ $? -eq 0 ]`